CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 [7.0.0-dev5]: https://github.com/microsoft/CCF/releases/tag/ccf-7.0.0-dev5
 
+### Added
+
+- Support for PreVote optimisation (#7419)
```

Comment:
This should say something like "Nodes respond to PreVote requests, but do not emit them", which would be clearer.